macros/PG.pl

```diff
@@ -185,18 +360,19 @@ ()
 	}
 }
 
-=head4 ADD_JS_FILE
+=head2 ADD_JS_FILE
 
-Request that the problem HTML page also include additional JS files
-from the webwork2/htdocs/ directory or from an external location.
+Request that the problem HTML page also include additional JavaScript files from
+the C<pg/htdocs> directory or from an external location.
 
 	ADD_JS_FILE($file, $external);
 
-If external is 1, it is assumed the full url is provided.  If external is 0 or
-not given, then file name will be prefixed with the webwork2/htdocs/ directory.
+If external is 1, it is assumed the full URL is provided. If external is 0 or
+not given, then file name will be served from the C<pg/htdocs> directory (if
+found).
 
 Additional attributes can be passed as a hash reference in the optional third
-argument.  These attributes will be added as attributes to the script tag.
+argument. These attributes will be added as attributes to the script tag.
```

Comment:
This existed before, but I think 5 lines down should have a `http:` or `https:` in front of the URL.

Reply:
I changed this to the link currently used by AppletObjects.pj which does include `https:`.  Thanks for pointing this out.